viam-cartographer/src/carto_facade/util.h

```diff
@@ -49,9 +49,8 @@ void write_float_to_buffer_in_bytes(std::string &buffer, float f);
 
 void write_int_to_buffer_in_bytes(std::string &buffer, int d);
 
-std::tuple<bool, cartographer::sensor::TimedPointCloudData>
-carto_sensor_reading(std::string sensor_reading,
-                     int64_t sensor_reading_time_unix_milli);
+std::tuple<bool, cartographer::sensor::TimedPointCloudData> carto_lidar_reading(
```

Comment:
[q] Is there no associated `carto_imu_reading` function as the conversion is simple enough to be done inline?

Reply:
Yes